demo/kubernetes/rook-storageclass.yaml

```diff
@@ -1,3 +1,18 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
```

Comment:
thats okai. For example, if we have two rook clusters one in namespace `rook1` and one in `rook2`, we can have rookpool resource named `mypool` created on namespace `rook1` and rookpool resource also named `mypool` in namespace `rook2`.  The rook-operators from each namespace will watch its corresponding pool.